_board/uchip.md

```diff
@@ -47,6 +47,13 @@ You can program uChip using virtually any IDE, in many languages (CircuitPython
 * Standards narrow-DIP footprint: 0.3” (7.62 mm) row spacing, 0.1” (2.54) pin spacing.
 * Pinout standard logic CMOS compatible: power and GND are on pin 16 and 8, so you can also emulate some 16 pin CMOS ICs (4000 and 74HC series)!  
 
+**Additional Notes:**
+
+* The uChip does not have an RGB LED.
```

Comment:
IMHO, we shouldn't mention hardware elements that are missing. The "features list" says that there is a LED.

Reply:
see previous comment